index.html

```diff
@@ -4,36 +4,57 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Portafolio</title>
 </head>
-
 <body>
     <header>
      <section class="home">
     <div class="cover">
-        <div class="bar-status">
-            <time> 9:41</time>
-            <img src="statusphone.jpg" alt="statusphone">
+        <div class="status">
+        <div class="status-mobile">
+            <time datetime="2022-08-22"> 9:41</time>
+        <div>
+        <!--phone signal-->
+        <i class="bi bi-reception-4"></i>
+        <!--wifilogo-->
+        <i class="bi bi-wifi"></i>
+        <!--battery file-->
+        <i class="bi bi-battery-full"></i>
         </div>
-        <div class="main">
-            <p>cesar valencia</p>
-            <button>hamburger</button>
+    </div>
+        <div class="nav-bar">
+            <p class="name">Cesar Valencia</p>
```

Comment:
Kindly, your name must contain an "A" tag as a link to the home page.

Reply:
i got a question, if apply the A tag, the redirection of this tag is linkein web or github page

Reply:
by convention, the main logo is directed to the homepage of your website

Reply:
ok i did it